### DISCUSSION TITLE
Broardcast goes offline and back on

### DISCUSSION THREAD

**Discussion by saitauthao** [ant-media/Ant-Media-Server#5024] [Q&A] [answered]
I set up a playlist to broadcast, and have 1 video setup to be looped from the playlist. At the end of the video and starting the loop again, the broadcast status goes offline for 5secs and back on. 

Any reasons for this? Do I need a specific encoding video? The video broadcast just fine, its when the video ends and starting the loop again it goes offline and back on. Making it not so smooth for looping.

Where I notice this was, broadcasting to youtube. The video would buffer at the end of the video and play the start again.

I am using the community edition of ant media server.

**Comment by yashtandon113** [comment]
@saitauthao 
Thanks for your question.

I uploaded two VoDs to my local server and created a playlist, but the broadcast did not go offline when I switched to the second video. Could you please confirm the scenario and the AMS version you are using?

**Reply by saitauthao** [reply, marked as answer]
thanks for replying.

oddly after a few restart, and stopped doing the buffer. so i am good now. 

thanks.
